### ISSUE TITLE
display ballot in rv2020 format

### ISSUE THREAD

**Issue by Bill-Keuntje** [rchain-community/rgov#153]

**Comment by Bill-Kunj** [comment]
@pfabre31 Is there a branch for this?

**Comment by David405** [comment]
Yeah, that happens to be ```ballot-ui```. I paused work on this temporarily cause of the conflicting standards, now I am a little clear on that, I will resume work.
